.circleci/config.yml

```diff
@@ -169,6 +169,67 @@ jobs:
       - codecov/upload:
           flags: 'test_<< parameters.old_pg_major >>_<< parameters.new_pg_major >>,upgrade'
 
+  test-custom:
```

Comment:
Once we agree on the naming, I will update the description, job name etc

Reply:
I guess we can change this now?